lib/utils/sdp-utils.js

```diff
@@ -12,19 +12,19 @@ const mergeSdpMedia = (sdp1, sdp2) => {
   return sdpTransform.write(parsedSdp1);
 };
 
+const getCodecPlacement = (sdp, codec) => sdpTransform.parse(sdp).media[0].rtp.findIndex((e) => e.codec === codec);
+
 const isOpusFirst = (sdp) => {
-  const parsedSdp = sdpTransform.parse(sdp);
-  const idx = parsedSdp.media[0].rtp.findIndex((entry) => entry.codec === 'opus');
-  return idx === 0;
+  return getCodecPlacement(sdp, 'opus') === 0;
 };
 
-const opusFirst = (sdp) => {
+const makeOpusFirst = (sdp) => {
   const parsedSdp = sdpTransform.parse(sdp);
   // Find the index of the OPUS codec
   const opusIndex = parsedSdp.media[0].rtp.findIndex((entry) => entry.codec === 'opus');
```

Comment:
I dont understand why we arent calling `getCodecPlacement` here?

Reply:
because we use parsedSdp result, calling getCodecPlacement here will make the code parse SDP 2 times for nothing

Reply:
I see.  Ok then perhaps change the function to accept a parsed SDP result instead of the raw SDP